apps/root/src/common/components/wallet-selector/index.tsx

```diff
@@ -392,15 +405,9 @@ const WalletSelector = ({ options, size = 'small' }: WalletSelectorProps) => {
 
   if (!wallets.length) {
     return (
-      <OptionsMenu
-        options={[connectWalletOption]}
-        mainDisplay={intl.formatMessage(
-          defineMessage({
-            defaultMessage: 'Connect your wallet',
-            description: 'connectWallet',
-          })
-        )}
-      />
+      <Button onClick={onConnectWallet} variant="text" size="small">
```

Comment:
I think this would look better with padding in 4px, to match the styles of the connected state

Reply:
Yeah this is just a test for now